demo/docker-compose.yml

```
@@ -15,7 +15,7 @@ services:
       - ./certs:/etc/ssl/certs
 
   tiles:
-    image: ghcr.io/maplibre/martin
+    image: ghcr.io/maplibre/martin:0.20.2
```

seems risky, lets revert this part

```suggestion
    image: ghcr.io/maplibre/martin
```